README.md

```diff
@@ -277,7 +277,7 @@ class { 'collectd::plugins::docker':
 }
```

Comment:
The readme instructions for apache will also need to be updated to use modules instead of instances.  Current instructions fail on test.